book/src/proposals/broadmoor.md

```diff
@@ -594,4 +618,7 @@ what you'd like to see.
 
 [^sdot safety study]: Seattle Department of Transportation's safety study:
 [City of Seattle Bicycle and Pedestrian Safety Analysis Phase&nbsp;2](<https://www.seattle.gov/documents/Departments/SDOT/VisionZero/SDOT_Bike%20and%20Ped%20Safety%20Analysis_Ph2_2420(0).pdf>)
+
+[^elevation data]:
```

Comment:
<img width="351" alt="Screen Shot 2021-07-08 at 4 38 15 PM" src="https://user-images.githubusercontent.com/217057/125002969-f59e0200-e00a-11eb-8c6d-00dad6faf398.png">


I guess the ordering here needs to reflect the ordering they appear in the article, so put elevation data first.

Reply:
Good catch! Fixed